beta-src/src/state/game/game-api-slice.ts

```diff
@@ -293,13 +305,116 @@ const drawBuilds = (state) => {
   }
 };
 
+const updateUnitsDisbanding = (state) => {
+  const {
+    data: {
+      data: { contextVars, currentOrders, units },
+    },
+    territoriesMeta,
+    ordersMeta,
+  }: {
+    data;
+    territoriesMeta;
+    ordersMeta;
+  } = current(state);
+  if (currentOrders && contextVars) {
+    const { context } = contextVars;
+    if (context.phase === "Retreats") {
+      const tMeta: TerritoriesMeta = territoriesMeta;
+      const terrMetaEntries = Object.entries(tMeta);
+      const overtakenTerritories = terrMetaEntries.filter(([id, val]) => {
+        return (
+          val.countryID === context.countryID &&
+          val.countryID !== val.ownerCountryID
```

Comment:
Can you explain how this is `overtakenTerritories`? This is filtering out the user's home units that are not currently owned by that user. Is this how you are checking that a unit is dislodged?

Reply:
This is one step of the process this paired with the units, currentOrders, and ordersMeta data allows me to get the disbanded units